stop.sh

```diff
@@ -1,8 +1,6 @@
 #!/bin/bash
 
 echo "Shutting down Akeneo service"
-source .env
-export SOURCE_PATH="${WORKING_DIR}"
 docker-compose down
```

Comment:
I get your point. But remember this is so optional. Just convenience call around. longer lines. :smile: